newIDE/app/src/EventsFunctionsExtensionEditor/index.js

```diff
@@ -37,6 +37,8 @@ import Mark from '../UI/CustomSvgIcons/Mark';
 import newNameGenerator from '../Utils/NewNameGenerator';
 const gd: libGDevelop = global.gd;
 
+export type ExtensionItemConfigurationAttribute = 'type' | 'visibility';
```

Comment:
Type is a bit vague, wdyt about `synchronicity` instead?

Reply:
'type' means: action, condition or expression.
I guess there is a parameter to avoid the view to kind of asking for or `forceUpdate` of their parent them selves, but I think that giving the exact attribute is going too far. I wanting to find a term that would work for anything that is showing in the tree but couldn't.
In the case of the synchronicity it can fell in the function type. For instance an asynchronious action. It avoid to add a 3rd value considering that we'll do the same thing in 3 cases.

Reply:
I think the 3rd value should be added because this:
<img width="345" alt="image" src="https://github.com/4ian/GDevelop/assets/32449369/38cb4a63-b286-4c83-b3dc-1e13d5769d02">
looks like a bug or someone forgot to change the attribute to the good one.
If I have to work on this part of the code, I will for sure feel like I want to fix something.

Reply:
ok, I'll do the change